index.html

```diff
@@ -197,26 +210,21 @@ <h2>Software projects</h2>
         make it possible to experiment with the proposed specifications:
       </p>
       <ul>
-        <li><a href="https://github.com/Maps4HTML/Web-Map-Custom-Element">Web map custom element</a>
-          <p>
-            A recreation of the HTML map element specification,
-            with MapML support,
-            an an HTML custom element,
-            using Leaflet to display the maps.
-          </p>
-          <p>
-            Note: The code currently uses the “version 0” custom elements API,
-            and therefore requires the Polymer polyfills to work.
-          </p>
-        </li>
-        <li><a href="https://github.com/Maps4HTML/MapML-Leaflet-Plugin">MapML Leaflet Plugin</a>
+        <li><a href="https://github.com/Maps4HTML/Web-Map-Custom-Element">Web-map custom element</a>
           <p>
-            The code that allows Leaflet to support MapML layers.
+            A polyfill of the proposed HTML map / layer element specification,
+            with MapML support, as a set of HTML custom elements,  using Leaflet
+            as the map rendering engine.
           </p>
         </li>
-        <li><a href="https://github.com/Maps4HTML/MapMLServer">MapML Server</a>
+        <li><a href="https://github.com/Maps4HTML/geoserver">GeoServer MapML MapML Community Module</a>
```

Comment:
Should this say "GeoServer MapML MapML Community Module"?

Reply:
Lol no, thanks, I'll fix it Thanks for your review.